newspack-nelson/sass/style.scss

```
@@ -262,6 +262,7 @@ h4,
 // Accent headers
 
 .accent-header,
+.widgettitle,
```

These specific styles are only needed on the `.widgettitle` class in the sidebar:

```suggestion
#secondary .widgettitle,
```